scripts/zones/Qulun_Dome/IDs.lua

```diff
@@ -34,6 +34,7 @@ zones[xi.zone.QULUN_DOME] =
     },
     npc =
     {
+        BEASTMENS_BANNER = 17383484,
```

Comment:
Just for some shift-proofing, GetFirstID('Beastmens_Banner')

Reply:
Ooh, works for me!